SingularityUI/app/actions/api/users.es6

```diff
@@ -0,0 +1,36 @@
+import { buildApiAction, buildJsonApiAction } from './base';
+
+export const FetchUserSettings = buildApiAction(
+  'FETCH_USER_SETTINGS',
+  userId => ({
+    url: `/users/settings?userId=${userId}`,
+    catchStatusCodes: [404] // Remove this once the API doesn't 404 when user has nothing set
+  })
+);
+
+export const UpdateUserSettings = buildJsonApiAction(
+  'UPDATE_USER_SETTINGS',
+  'POST',
+  (userId, settings) => ({
+    url: `/users/settings?userId=${userId}`,
```

Comment:
I like that, it'll make the whole settings API simpler.
What status code will be returned if users aren't supported? I'll have to ignore that code.